package.json

```diff
@@ -16,6 +16,7 @@
 		"lint": "eslint src",
 		"jsdoc": "rm -rf ./doc && jsdoc -c jsdoc.json",
 		"coverage": "karma start --coverage",
+    "test:node": "node ./dist/persist.js",
```

Comment:
There seems to be confusion because the term test is generally used as a command or prefix to run unit/e2e test.
The purpose is to simply determine whether there is a problem with initialization when the module is executed in node env.
I think names like simulate:node, emulate:node would be more appropriate.